src/Godot.Bindings/Bridge/GodotBridge.cs

```diff
@@ -78,6 +78,15 @@ private static unsafe void Initialize(GetProcAddressFunction getProcAddress, voi
             minimum_initialization_level = (GDExtensionInitializationLevel)configuration.MinimumInitLevel,
         };
 
+        GodotSynchronizationContext.InitializeSynchronizationContext();
+
+        var mainLoopCallbacks = new GDExtensionMainLoopCallbacks()
+        {
+            frame_func = Marshal.GetFunctionPointerForDelegate(Frame_Native),
```

Comment:
It's preferable to get the pointer using the `&` operator. The Marshal API should not be used because, from what I understand, it needs to create a wrapper which adds unnecessary overhead.

```suggestion
            frame_func = (nint)(delegate* unmanaged[Cdecl]<void>)(&Frame_Native),
```